README.md

```diff
@@ -38,27 +38,31 @@ This tap:
 
 ## Running the tap
 
-- Create config file with the following variables:
-    - consumer_key
-    - client_secret
-    - start_date
+1) Create config file with the following variables:
 ```
 {
-  "consumer_key": "REPLACE",
-  "client_secret": "REPLACE",
-  "start_date": "2016-01-01"
+  "client_id": "SECRET",
+  "client_secret": "SECRET",
+  "tenant_id": "SECRET",
+  "refresh_token": "GENERATED BY NOTEBOOK",
+  "start_date": "2000-01-01"
 }
 ```
-- Validate that the private key is located at the following path: `~/.ssh/privatekey.pem`. The key is generated for the Xero application using Xero's developers portal. More details: https://developer.xero.com/documentation/auth-and-limits/private-applications
 
-Execute the following commands:
+1.1) The secrets are all stored in 1Password in the Analytics DB Credentials Vault
 
-```bash
-pip install -e .
-tap-xero -c config.json -d > properties.json
-tap-xero -c config.json -p properties.json >> output.json
-```
+1.2) Using those secrets and the jupyter notebook called `get_initial_xero_tokens`, run through the web-based auth process to get your initial refresh token. Using this token the tap will then chain your tokens and will be able to run on its own. You must use the notebook manually before every run of the tap. See more on this OAuth protocol [here](https://developer.xero.com/documentation/oauth2/auth-flow).
+
+2) Run `poetry update` to ensure dependencies are up to date
```

Comment:
`poetry update` failed on my machine with the following error but I was still able to run the next steps
```Updating dependencies
Resolving dependencies... (0.9s)

[SolverProblemError]
Because dia-tap-xero depends on dia-target-postgres (0.0.6) which doesn't match any versions, version solving failed.```

Reply:
this is fixed now

Reply:
I'm still getting the same issue

Reply:
This is very odd because it exists here:
https://manage.fury.io/dashboard/dialogue/package/pkg_Nmhjw/version/ver_2acI8E